relay-server/src/services/outcome.rs

```diff
@@ -603,7 +604,19 @@ impl DiscardItemType {
             Self::Event => "event",
             Self::Transaction => "transaction",
             Self::Security => "security",
-            Self::Attachment => "attachment",
+            // TODO: There might be a better way to do this, but I did not yet find it.
+            Self::Attachment(DiscardAttachmentType::Attachment) => "attachment:attachment",
+            Self::Attachment(DiscardAttachmentType::Minidump) => "attachment:minidump",
+            Self::Attachment(DiscardAttachmentType::AppleCrashReport) => {
+                "attachment:apple_crash_report"
+            }
+            Self::Attachment(DiscardAttachmentType::EventPayload) => "attachment:event_payload",
+            Self::Attachment(DiscardAttachmentType::Breadcrumbs) => "attachment:breadcrumbs",
+            Self::Attachment(DiscardAttachmentType::Prosperodump) => "attachment:prosperodump",
+            Self::Attachment(DiscardAttachmentType::UnrealContext) => "attachment:unreal_context",
+            Self::Attachment(DiscardAttachmentType::UnrealLogs) => "attachment:unreal_logs",
+            Self::Attachment(DiscardAttachmentType::ViewHierarchy) => "attachment:view_hierarchy",
+            Self::Attachment(DiscardAttachmentType::Unknown) => "attachment:unknown",
```

Comment:
I am not super happy with this but did not yet find a way to merge a static str and a variable holding a static str into a static str.